vscode-dotnet-runtime-extension/package.json

```diff
@@ -9,8 +9,8 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft Corporation",
-	"displayName": ".NET Install Tool for Extension Authors",
-	"description": "Allows acquisition of the .NET runtime specifically for VS Code extension authors.",
+	"displayName": ".NET Runtime Install Tool",
```

Comment:
The display name was changed but in the screenshot you provided it appears it did not get reflected in the extension page?

Reply:
![image](https://github.com/dotnet/vscode-dotnet-runtime/assets/573979/482e4782-6edc-4c44-a39c-9dbf09926327)

My fault - I reused the same version number when testing locally and so VSCode didn't update all of the window dressing. When I tested with a different version it all looks correct.